Meditation/Meditation/View/Home/SleepingRow.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+struct SleepingRow: View {
+    var categoriesSleeping = ["Sweet Sleep", "Insomnia", "Depressic", "Disorder"]
```

Comment:
I'd say this is rather `mood` or smth like that. Can we represent this with `enum`? Ping Vlad for some hints on how to bind localized strings to `enum`.